src/DependencyInjection/SonataDoctrineORMAdminExtension.php

```diff
@@ -59,5 +60,21 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->getDefinition('sonata.admin.builder.orm_show')
             ->replaceArgument(1, $config['templates']['types']['show']);
+
+        // NEXT_MAJOR: remove this block.
+        $deprecatedEmptyFilterDefinition = $container->getDefinition('Sonata\DoctrineORMAdminBundle\Filter\EmptyFilter');
```

Comment:
Could we use `EmptyFilter::class`?

Reply:
This would trigger autoloading the deprecated class :see_no_evil: I could try it. But does it matter?

Reply:
>This would trigger autoloading the deprecated class

IMO, it should not.
AFAIK, you can use reference even to nonexistent classes.
>I could try it. But does it matter

I think using the class reference instead of a string helps the SCA tools to detect and prevent issues.

Reply:
yeah should work indeed :+1: let's try it